crates/astria-conductor/src/executor/mod.rs

```diff
@@ -241,32 +242,29 @@ pub(crate) struct Executor {
     /// Required to mark firm blocks received from celestia as executed
     /// without re-executing on top of the rollup node.
     blocks_pending_finalization: HashMap<u32, Block>,
+
+    /// The maximum permitted spread between firm and soft blocks.
+    max_spread: Option<usize>,
 }
 
 impl Executor {
-    #[instrument(skip_all)]
+    #[instrument(skip_all, err)]
```

Comment:
what does the `err` addition do?

Reply:
Sorry, overlooked this: this generates an error event under the span created by the instrumented method.